libmscore/album.cpp

```diff
@@ -0,0 +1,248 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "album.h"
+#include "score.h"
+#include "xml.h"
+#include "musescoreCore.h"
+
+namespace Ms {
+
+//---------------------------------------------------------
+//---------------------------------------------------------
+//
+//   AlbumItem
+//
+//---------------------------------------------------------
+//---------------------------------------------------------
+
+AlbumItem::AlbumItem(Album* album)
+{
+    this->album = album;
+}
+
+AlbumItem::AlbumItem(Album* album, MasterScore *score, bool enabled) : AlbumItem(album)
+{
+    std::cout << "New album item..." << std::endl;
```

Comment:
Use `qDebug`.

Reply:
I will probably delete it when I reach an acceptable level of stability.

Reply:
If you use `qDebug` you probably won't have to delete them :)

Reply:
Hmmm no, final `qDebug`s don't need these details. So fine, you can use `std::cout`s and delete them when they aren't needed anymore.